packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-function-settings.test.tsx

```diff
@@ -836,32 +804,6 @@ describe("Locked Function Settings", () => {
                 });
             });
 
-            test("aria label does not auto-generate with domain when both values are null", async () => {
-                // Arrange
-                const onChangeProps = jest.fn();
-                render(
-                    <LockedFunctionSettings
-                        {...defaultProps}
-                        ariaLabel={undefined}
-                        onChangeProps={onChangeProps}
-                        domain={[null, null]}
-                    />,
-                    {wrapper: RenderStateRoot},
-                );
-
-                // Act
-                const autoGenButton = screen.getByRole("button", {
-                    name: "Auto-generate",
-                });
-                await userEvent.click(autoGenButton);
-
-                // Assert
-                expect(onChangeProps).toHaveBeenCalledWith({
-                    ariaLabel:
-                        "Function with equation y=x^2. Appearance solid gray.",
-                });
-            });
-
```

Comment:
This test isn't needed anymore either. We have a similar test for the case where the `domain` is `[-Infinity, Infinity]`.